frontend/pages/admin/UserManagementPage/components/UserForm/UserForm.tsx

```diff
@@ -347,11 +370,15 @@ class UserForm extends Component<ICreateUserFormProps, ICreateUserFormState> {
             <img src={OpenNewTabIcon} alt="open new tab" />
           </a>
         </InfoBanner>
-        <SelectedTeamsForm
-          availableTeams={availableTeams}
-          usersCurrentTeams={teams}
-          onFormChange={onSelectedTeamChange}
-        />
+        {availableTeams.length !== 0 ? (
```

Comment:
Small thing. Since we're comparing numbers, I think we should stick to doing `length > 0`, since the length of an array will never be less than zero.